packages/ddp-server/livedata_server.js

```diff
@@ -639,7 +645,7 @@ Object.assign(Session.prototype, {
           return true;
         });
 
-        if (_.has(self.protocol_handlers, msg.msg)) {
+        if (has(self.protocol_handlers, msg.msg)) {
```

Comment:
Could this be something like `msg.msg in self.protocol_handlers`?

Reply:
This seems to be pretty new syntax. Here is some documentation for folk: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in